From: Ole Streicher <olebole@debian.org>
Date: Sat, 24 Mar 2012 17:21:55 +0100
Subject: Use tcllib instead of packages snit

Use the snit package included into tcllib bundle instead of private
copies of snit*.tcl
---
 hv/hv3_browser.tcl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hv/hv3_browser.tcl b/hv/hv3_browser.tcl
index 461a4b9..0a0ebdb 100644
--- a/hv/hv3_browser.tcl
+++ b/hv/hv3_browser.tcl
@@ -2,6 +2,7 @@
 
 package require sqlite3
 package require Tkhtml 3.0
+package require snit
 
 proc sourcefile {file} [string map              \
   [list %HV3_DIR% [file dirname [info script]]] \
@@ -9,7 +10,6 @@ proc sourcefile {file} [string map              \
   return [file join {%HV3_DIR%} $file] 
 }]
 
-source [sourcefile snit.tcl]
 source [sourcefile hv3_widgets.tcl]
 source [sourcefile hv3_notebook.tcl]
 source [sourcefile hv3_db.tcl]
